config/github-actions/ci/run_step.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+case "$1" in 
+
+  # Configure qmcpack using cmake out-of-source builds 
+  configure)
+   
+    cd ${GITHUB_WORKSPACE}/..
+    mkdir qmcpack-build
+    cd qmcpack-build
+    cmake -GNinja -DCMAKE_INSTALL_PREFIX=${GITHUB_WORKSPACE}/../ncio-install ${GITHUB_WORKSPACE}
+    ;;
+
+  # Build using ninja
+  build)
+    cd ${GITHUB_WORKSPACE}/../qmcpack-build
+    ninja
+    ;;
+
+  # Run unit and deterministic tests
+  test)
+    cd ${GITHUB_WORKSPACE}/../qmcpack-build
+    ctest -R unit
+    ctest -R deterministic -LE unstable
```

Comment:
please replace
```
    ctest -R unit
    ctest -R deterministic -LE unstable
```
with
```
    ctest -L deterministic
```
namely by label instead of test name.

Reply:
@ye-luo good catch, thanks!